example_notebooks/example_scripts/example_dp.py

```diff
@@ -26,20 +26,20 @@
 
 idr_id = "idr0013"
 tmp_dir = pathlib.Path("../tmp/")
-final_data_dir = pathlib.Path("../mitocheck_control_features/DP_features")
+final_data_dir = pathlib.Path("../example_extracted_features/DP_features")
 try:
     shutil.rmtree(tmp_dir)
     # shutil.rmtree(final_data_dir)
 except:
     print("No files to remove")
 
-stream = DeepProfilerRun(idr_id, tmp_dir, final_data_dir, log='dp_idrstream.log')
+stream = DeepProfilerRun(idr_id, tmp_dir, final_data_dir, log='example_logs/dp_idrstream.log')
 
 
 # In[4]:
 
 
-aspera_path = pathlib.Path("/home/jenna/.aspera/ascli/sdk/ascp")
+aspera_path = pathlib.Path("/home/roshankern/.aspera/ascli/sdk/ascp")
```

Comment:
Depending on how compatible this may be, consider using a tilde (`~`) here to reference the home directory on any system instead of using the absolute path for specific users. (also applies to other areas in this file)
```suggestion
aspera_path = pathlib.Path("~/.aspera/ascli/sdk/ascp")
```

Reply:
Done in https://github.com/WayScience/IDR_stream/pull/15/commits/098c83c22189ecbd5ac1f38b231a33fc499e9ff4